### PR TITLE
Add updates from doing live tutorial demos.

### PR DESCRIPTION
**Pull request by jroesch**
This adds some changes from my working branch for the tutorials.

### CODE REVIEW

**Comment by jroesch** [comment]
cc @anwang2009 and @vanessahlyan

File: tutorials/vision_accel/t [hunk 1]
```diff
@@ -0,0 +1 @@
+./deploy_to_eks.sh bert 186900524924_Sandbox-Developer bertv1 186900524924.dkr.ecr.us-west-2.amazonaws.com octoml-sandbox-sxq590bv us-west-2 8080
```
Comment (nibalizer) [inline tutorials/vision_accel/t:1]: What's this?

File: README.md [hunk 2]
```diff
@@ -11,36 +11,46 @@ and K8s cluster, but the containers can be used in any environment that supports
 We support TensorFlow SavedModel, TensorFlow GraphDef, Torchcript (PyTorch), and ONNX models.
 We support x86 and CUDA machines. We do not yet support ARM (including M1 Macs).
 
+For more information about versioning, architecture, and other compatibility questions see [our roadmap](roadmap.md).
```
[the remark this answers is not in the document]
Comment (jroesch) [reply README.md:14]: I think these changes modulo roadmap are good.

[the remark this answers is not in the document]
Comment (jroesch) [reply README.md:14]: I think these changes modulo roadmap are good.

File: tutorials/README.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ For example:
 
 Finally, run inferences on the container deployed to EKS
 
-The last step of the `deploy_to_eks.sh` script port-forwards the EKS deployment's triton GRPC endpoint to localhost
+The last step of the `deploy_to_eks.sh` script port-forwards the EKS deployment's tvriton GRPC endpoint to localhost
```
Comment (jroesch) [inline tutorials/README.md:186]: This is definitely not a good change.

Comment (jroesch) [inline tutorials/README.md:186]: This is definitely not a good change.

File: tutorials/deploy_to_eks.sh [hunk 4]
```diff
@@ -30,21 +31,22 @@ fi
...
-helm install demo ./helm_chart \
+helm install $MODEL_NAME ./helm_chart \
 	--set image.tag=$MODEL_NAME:$DOCKER_IMAGE_TAG \
 	--set image.repository=$AWS_REGISTRY_URL \
+	--set Name=$MODEL_NAME \
         --timeout 5m0s --wait --atomic
 
 # port forward
-kubectl port-forward service/demo -n demo 8080:80
+kubectl port-forward service/$MODEL_NAME -n $MODEL_NAME $DEST_PORT:80
```
Comment (jroesch) [inline tutorials/deploy_to_eks.sh:52]: These are good changes brought over from @anwang2009's examples changes.

Comment (jroesch) [inline tutorials/deploy_to_eks.sh:52]: These are good changes brought over from @anwang2009's examples changes.

File: tutorials/helm_chart/templates/deployment.yaml [hunk 5]
```diff
@@ -2,16 +2,16 @@ apiVersion: apps/v1
 kind: Deployment
```
Comment (jroesch) [inline tutorials/helm_chart/templates/deployment.yaml:2]: Same with these changes from @anwang2009 chart changes.

Comment (jroesch) [inline tutorials/helm_chart/templates/deployment.yaml:2]: Same with these changes from @anwang2009 chart changes.

File: tutorials/vision_accel/t (hunk 1 above)
Comment (jroesch) [reply tutorials/vision_accel/t:1]: DELETE THIS

Comment (jroesch) [reply tutorials/vision_accel/t:1]: DELETE THIS

**Comment by nibalizer** [comment]
Closing in favor of #13 cc @jroesch

**Pull request closed without merging**